src/site/content/en/blog/reporting-api/index.md

```diff
@@ -0,0 +1,660 @@
+      </tr>
+      <tr>
+        <td><code>type</code></td>
+        <td>A report type, for example <code>csp-violation</code> or <code>coep</code>.</td>
+      </tr>
+      <tr>
+        <td><code>url</code></td>
+        <td>The address of the document or worker from which the report was generated. Sensitive data such as username, password, and fragment are <a href="https://w3c.github.io/reporting/#capability-urls">stripped<a> from this URL.</td>
+      </tr>
+      <tr>
+        <td><code>user_agent</code></td>
+        <td>User-Agent header of the request from which the report was generated.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+{% Details %}
+
+{% DetailsSummary 'h4' %}
+
+Example list of reports
+
+{% endDetailsSummary %}
+
+```json
+[
+  {
+    "age": 420,
+    "body": {
+      "columnNumber": 12,
+      "disposition": "enforce",
+      "lineNumber": 11,
+      "message": "Document policy violation: document-write is not allowed in this document.",
+      "policyId": "document-write",
+      "sourceFile": "https://site.example/script.js"
+    },
+    "type": "document-policy-violation",
+    "url": "https://site.example/",
+    "user_agent": "Mozilla/5.0... Chrome/92.0.4504.0"
+  },
+  {
+    "age": 510,
+    "body": {
+      "blockedURL": "https://site.example/img.jpg",
+      "destination": "image",
+      "disposition": "enforce",
+      "type": "corp"
+    },
+    "type": "coep",
+    "url": "https://dummy.example/",
+    "user_agent": "Mozilla/5.0... Chrome/92.0.4504.0"
+  }
+]
+```
+
+{% endDetails %}
+
+{% Aside %} Note:
+
+- In the new Reporting API (v1), thee browser only batches together reports in the same
+  request if they are generated by the same document (page). Reports are not sent in the
+  same request if they come from different origins, or even different documents on the
+  same origin such as different pages on the same website. Learn more in the [migration
+  guide](/reporting-api-migration).
+- In the new Reporting API, reports are scoped to **documents**. This means that for one
+  given origin, different documents (`site.example/page1` and `site.example/page2`) can
+  send reports to different endpoints. If you're migrating from the legacy Reporting API
+  to the new Reporting API, check the [migration guide](/reporting-api-migration).
+- Reports are sent with a delay to increase the chances that reports are sent in batches.
+  This saves bandwidth to be respectful to the user's network connection. This is
+  especially important on mobile.
+
+{% endAside %}
+
+## Reports you get
+
+Let's assume that you've set up `Reporting-Endpoints` and a few policies on your page.
+
+✅ Reports that are generated due to violations of these policies (or deprecations) on your
+page will be sent to the endpoint(s) you've configured. This includes violations committed
+by **third-party scripts** running on your page.
+
+❌ Violations committed by a **cross-origin iframe** on your page will **not** be reported
```

Comment:
Again, emoji weirdness for screen readers. It reads as "Cross mark violations committed by a cross origin iframe" which could be confusing. I think it's best to not use emoji in this way in our technical docs and limit the use to what makes sense "semantically" rather than just visually. (Not saying you shouldn't use emoji at all!)

Reply:
SG, removed these emojis. Thanks for checking!